games-rpg/flare/flare-1.14.recipe

```diff
@@ -0,0 +1,135 @@
+SUMMARY="Free/Libre Action Roleplaying Engine"
+DESCRIPTION="Flare is an open source, 2D action RPG licensed under the GPL3 license. Its game play \
+can be likened to the games in the Diablo series.
+
+While the core engine is written in C++ with SDL2, there is an easy to use modding interface, \
+by using it, it’s possible to create entirely new games. On the Flare website's Mods page, \
+you can find games made by the Flare team, as well a few select community projects.
+
+This package provides only the Flare engine, gameplay mods can be installed through \
+other packages or by putting them manually in 'non-packaged/data/flare/mods'."
+HOMEPAGE="https://flarerpg.org/"
+COPYRIGHT="2010-2022 Clint Bellanger & Flare team"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/flareteam/flare-engine/archive/v$portVersion/flare-engine-v$portVersion.tar.gz"
+CHECKSUM_SHA256="2c1bafeaa5ac26c10449bfcb7151b06e8a22547aa7364d2a39bbcb135aa50a09"
+SOURCE_DIR="flare-engine-$portVersion"
+SOURCE_URI_2="https://github.com/flareteam/flare-game/archive/v$portVersion/flare-game-v$portVersion.tar.gz"
+CHECKSUM_SHA256_2="65758462070aa88842084f8ee69083d8226e93cfbf83481663276d8307494b8e"
+SOURCE_DIR_2="flare-game-$portVersion"
+ADDITIONAL_FILES="flare.rdef.in"
```

Comment:
Why did you remove the old patchset?

Reply:
'cause it builds, runs and finds all the needed paths by itself even without the patch, so why keep it?

Reply:
Agree here with @korli it's not because it builds/runs without it should be thrown away, @threedeyes added the Haiku specific files/patches for a reason.

Reply:
Alright, will wait for feedback from the patch's creator, there may be more to that patch than I realized.